### ISSUE TITLE
Figure out a migration strategy

### ISSUE THREAD

**Issue by llimllib** [llimllib/gpt-bash-cli#7]
Right now I assume that I can just YOLO migrations because I'm ~the only user. If people actually start using the project, I should probably have a way to run migrations safely.

**Comment by simonmysun** [comment]
Hi, I have a question that doesn't seem out of place here: where do you expect user to install the bash scripts?

**Comment by llimllib** [comment]
@simonmysun it's just a bash script, so you can put it anywhere on your PATH

(`PATH` is an environment variable that tells your shell where to look for executables. You can do `echo $PATH` in your shell and it should print out a colon-separated list of directories; put this script in any of those.

I personally keep it in `~/.local/bin`, but that might not be a directory in your `PATH`

if you're on windows, most of this is not quite right and this script probably wouldn't work for you)

**Comment by simonmysun** [comment]
Yes, I understand that you can add it anywhere. I'm just thinking about letting [ell](https://github.com/simonmysun/ell) to read its configuration from `XDG_CONFIG_HOME`. I don't want to place the scripts into `/usr` if it is not managed by a package manager. 

Thank you for the suggestion, I think `~/.local/bin` is a good choice. I will guide user to download released version there. 

This is however not in [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/latest/) but in [FILE-HIERARCHY(7)](https://man.archlinux.org/man/file-hierarchy.7.en#USER_PACKAGES). I'm not sure if all the operating systems follows it.
